src/components/editions/styles.ts

```diff
@@ -0,0 +1 @@
+export const editionsArticleWidth = 457;
```

Comment:
This is a very specific number 🤔

Reply:
Indeed - was going of these designs: https://www.figma.com/file/AgfOrrLI78asahmA3Z9Env/AUS%2FUS-xtra-designs?node-id=1713%3A0

Is there a grid value we can use instead?

Reply:
Question for @prisalcalde @pradasa @benwuersching, is there a width from Source's grid that we could use for this?